frontend/packages/component-library/src/cms/snapshot/snapshot.module.scss

```
@@ -0,0 +1,48 @@
+@use '@code-dot-org/component-library-styles/colors.scss';
+@use '@code-dot-org/component-library-styles/mixins.scss' as mixins;
+@use '@code-dot-org/component-library-styles/typography.module.scss' as
+  typography;
+
+// SnapshotSection common styles
+.snapshot {
+  border-radius: 0.25rem;
```

You can use the `$regular-border-radius` variable here.
```suggestion
  border-radius: $regular-border-radius;
```